src/libraries/System.Diagnostics.Process/tests/ProcessModuleTests.cs

```diff
@@ -89,5 +93,33 @@ public void ModulesAreDisposedWhenProcessIsDisposed()
             process.Dispose();
             Assert.Equal(expectedCount, disposedCount);
         }
+
+        [ActiveIssue("https://github.com/dotnet/runtime/pull/335059")]
+        [ConditionalFact(nameof(AreAllLongPathsAvailable))]
```

Comment:
Does this fail if you directly consume the `PathFeatures` method, instead of creating a static one in this class?:

```suggestion
        [ConditionalFact(nameof(PathFeatures.AreAllLongPathsAvailable))]
```

Reply:
I didn't know that it's possible, but it turns out that you can specify `Type` and member name to the `ConditionalFact` ctor:

```cs
[ConditionalFact(typeof(PathFeatures), nameof(PathFeatures.AreAllLongPathsAvailable))]
```